Customer.DomainServices/Customer.DomainServices.csproj

```
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Customer.DomainModels\Customer.DomainModels.csproj" />
+    <ProjectReference Include="..\Customer.Infrastructure.Data\Customer.Infrastructure.Data.csproj" />
```

Como o Infrastructure.Data já contempla o DomainModels, não há necessidade de referenciá-lo aqui.
```suggestion
  <ItemGroup>
    <ProjectReference Include="..\Customer.Infrastructure.Data\Customer.Infrastructure.Data.csproj" />
```